src/main.rs

```diff
@@ -372,6 +373,14 @@ fn create_app<'a>(
                 .takes_value(true)
                 .possible_values(&["true", "false", "log"])
                 .default_value("true"),
+        )
+        .arg(
```

Comment:
Not convinced we need this. Do situations exist where both are available on the host?

Reply:
Not right now. If you want auto section at all times I can drop this argument and other code.

Reply:
I generally prefer to keep things simpler and if there is no benefit to adding it right now then I would prefer to drop it. We can always add it later.

Reply:
Sure. That's fine by me.